kinto/__main__.py

```diff
@@ -40,12 +46,25 @@ def main(args=None):
     config_file = args['ini_file']
 
     if args['which'] == 'init':
-        if not os.path.exists(config_file):
-            init(config_file)
-        else:
+        if os.path.exists(config_file):
             print("%s already exist." % config_file, file=sys.stderr)
             sys.exit(1)
 
+        backend = args['backend']
+        if not backend:
```

Comment:
I think I would keep that check in the input() methods when passing backend=None to the method.